runtime-modules/forum/src/mock.rs

```diff
@@ -624,11 +622,11 @@ pub fn generate_poll(
     }
 }
 
-pub fn generate_poll_timestamp_cases(
+pub fn generate_poll_input_timestamp_cases(
```

Comment:
Add comments for public methods.

Reply:
Done https://github.com/Joystream/joystream/pull/2531/commits/4bb08fdb7c9a5c49976eb313b279b62aa8973780